<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCR_TOOL</class>
 <widget class="QMainWindow" name="OCR_TOOL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCR_TOOL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>481</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="path">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="id_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="name_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="ad1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>AD1</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ad1_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="date_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="ad2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>AD2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ad2_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="open"/>
   </widget>
   <addaction name="menuFILE"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
